content/reference/api/hub/latest.yaml

```
@@ -892,6 +892,32 @@ paths:
           $ref: "#/components/responses/Forbidden"
         "404":
           $ref: "#/components/responses/NotFound"
+  /v2/repositories/{namespace}/{repository}/groups:
+    parameters:
+      - $ref: "#/components/parameters/namespace"
+      - $ref: "#/components/parameters/repository"
+    post:
+      summary: Assign a group (Team) to a repository for access
+      tags:
+        - repositories
+      security:
+        - bearerAuth: []
+      parameters:
+        - in: query
+          name: group_name
+          required: true
+          schema:
+            type: string
+          description: Name of the group (team) in the organization.
+        - in: query
+          name: permission
+          required: true
+          schema:
+            type: string
+          description: string - possible values : read, write, admin
```

```suggestion
            type: string
            description: |
              Access level for the group. Possible values:
              - `read`
              - `write`
              - `admin`
```

gives more context and aligns with formatting of other schemas